tests/integration/MLflow.Server.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:latest
+
+# Install python packages
+RUN pip install mlflow psycopg2 boto3
+COPY mlflow-setup.py .
+CMD mlflow server --backend-store-uri ${BACKEND_STORE_URI} --host 0.0.0.0 --port 5000
+
```

Comment:
we probably want to be able to target different versions of MLflow (and maybe even Python)